frontend/public/components/storage/create-pvc.tsx

```diff
@@ -0,0 +1,368 @@
+/* eslint-disable no-undef */
+import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+
+import { connectToPlural } from '../../kinds';
+import { k8sCreate, K8sKind, referenceFor } from '../../module/k8s';
+import { ButtonBar, history, LoadingBox, NameValueEditor, resourceObjPath, RequestSizeInput } from '../utils';
+import { ListDropdown } from '../RBAC/bindings';
+import { RadioInput } from '../radio';
+import { Checkbox } from '../checkbox';
+
+const StorageClassDropdown: React.SFC<StorageClassDropdownProps> = props => {
+  const kind = 'StorageClass';
+  const resources = [{ kind }];
+  const { selectedKey, required, namespace } = props;
+  return (
+    <ListDropdown
+      {...props}
+      desc="Storage Classes"
+      resources={resources}
+      selectedKeyKind={kind}
+      placeholder="Select storage class"
+      selectedKey={selectedKey}
+      required={required}
+      namespace={namespace}
+      name={name}
+    />
+  );
+};
+// This form is done a little odd since it is used in both its own page and as
+// a sub form inside the attach storage page.
+export class CreatePvcForm extends React.Component<CreatePvcFormProps, CreatePvcFormState> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      storageClass: '',
+      pvcName: '',
+      accessMode: 'ReadWriteOnce',
+      requestSizeValue: '1',
```

Comment:
Empty value by default?

Reply:
The old console defaulted to 1 GiB.  It seems like a good thing to do in case the user does not put in a value.